bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -130,6 +154,19 @@ pub fn config_builder(cipher_prefs: &security::Policy) -> Result<crate::config::
 
 type LocalDataBuffer = RefCell<VecDeque<u8>>;
 
+#[derive(Debug)]
+#[allow(dead_code)] // needed by renegotiate.rs tests
```

Comment:
I think this comment should follow the struct, and then maybe we no longer need to allow `dead_code` for the actual `TestPair`?
https://github.com/aws/s2n-tls/blob/b483357680577fdd5f94c14cbc4e92a7563f0160/bindings/rust/s2n-tls/src/testing.rs#L159-L161